extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -356,9 +355,11 @@ private static HttpServerOptions createSslOptions(HttpConfiguration httpConfigur
                     serverOptions);
         }
 
-        for (String cipher : sslConfig.cipherSuites.orElse(Collections.emptyList())) {
-            serverOptions.addEnabledCipherSuite(cipher);
-        }
+        sslConfig.cipherSuites.ifPresent(cipherSuites -> {
```

Comment:
We try to avoid lambdas in runtime code as it consumes more memory than strictly necessary.